test/src/main/resources/af00251.xml

```diff
@@ -92,7 +92,12 @@
                     <ns5:displayName>Deltid</ns5:displayName>
                 </ns5:cv>
             </ns3:delsvar>
-            <ns3:delsvar id="2.3">22</ns3:delsvar>
+            <ns3:delsvar id="2.3">
+                <ns5:pq>
+                    <ns5:value>22.0</ns5:value>
+                    <ns5:unit>hour</ns5:unit>
```

Comment:
hour -> h

Reply:
Enligt kravet ska unit vara "hour". https://inera-certificate.atlassian.net/wiki/spaces/EIT/pages/1179877494/Intygsspecifikation+L+karintyg+f+r+deltagare+i+arbetsmarknadspolitiska+program+med+aktivitetsst+d+utvecklingsers+ttning+eller+etableringsers+ttning+AF00251+v.1.x

Reply:
OK! Men det bör isåfall ändras. Kravet följer inte kravet i RIV-TA.